### PR TITLE
Lint

### PR DESCRIPTION
**Pull request by Eabramov84**

### CODE REVIEW

File: dataprovider/dataprovider_test.go [hunk 1]
```diff
@@ -298,7 +298,7 @@ func TestGetData(t *testing.T) {
 		}
 	*/
 
-	data, err = provider.GetData("Body.Flux.Capacitor", nil)
+	_, err = provider.GetData("Body.Flux.Capacitor", nil)
```
Comment (al1img) [inline dataprovider/dataprovider_test.go:301]: Combine in one line.

File: plugins/telemetryemulatoradapter/telemetryemulatoradapter.go [hunk 2]
```diff
@@ -264,18 +264,18 @@ func (adapter *TelemetryEmulatorAdapter) getDataFromTelemetryEmulator() (visData
...
+		data, err := adapter.getDataFromTelemetryEmulator()
+		if err != nil {
+			log.Errorf("Can't read data: %s", err)
+			continue
 		}
+		if err = adapter.baseAdapter.SetData(data); err != nil {
+			log.Errorf("Can't update data: %s", err)
+			continue
+		}
+
```
Comment (al1img) [inline plugins/telemetryemulatoradapter/telemetryemulatoradapter.go:278]: Remove empty line

File: dataprovider/dataprovider_test.go (hunk 1 above)
Comment (Eabramov84) [reply dataprovider/dataprovider_test.go:301]: done

File: plugins/telemetryemulatoradapter/telemetryemulatoradapter.go (hunk 2 above)
Comment (Eabramov84) [reply plugins/telemetryemulatoradapter/telemetryemulatoradapter.go:278]: removed

**Pull request merged**